tests/Integration/tests/configuration/expected/objects.inv.json

```diff
@@ -0,0 +1,18 @@
+{
+    "std:doc": {
```

Comment:
Could you explain what the difference is of std:doc and std:label, because the contain the same values?

Reply:
In a rst project consisting of several documents each document is listed in std:doc it can then be used to link to a document using its path. i.e. ``:doc:`index` ``

std:label is linking to each headline (we only have one in this example). If there is more then one headline in one Document=rst file you can link to each headline. you can also link to elements that have an explicit anchor defined.

The file is used when we are linking between different manuals / projects.

It also contains the version and manual/project title that was set in the guides.xml, that is why I included it in the tests

Reply:
Ah, I see. Those would not be part of the settings.cfg migration, or are they?!

Reply:
No, they are output and not input